modules/lightmapper_rd/lm_compute.glsl

```diff
@@ -76,6 +76,8 @@ layout(set = 1, binding = 3) uniform DenoiseParams {
 	float albedo_bandwidth;
 	float normal_bandwidth;
 
+	int half_search_window;
+
```

Comment:
Same reason as before, remove the empty line to pack together the 16 byte segments.